bochs/configure.ac

```diff
@@ -1252,6 +1252,29 @@ if test "$support_cet" -gt 0 -a "$use_x86_64" = 0; then
   AC_MSG_ERROR([Control Flow Enforcement Technology feature emulation require x86-64 enabled])
 fi
 
+support_uintr=0
+AC_MSG_CHECKING(for User Level Interrupt support)
+AC_ARG_ENABLE(cet,
+  AS_HELP_STRING([--enable-uintr], [for User Level Interrupt support (no)]),
+  [if test "$enableval" = yes; then
+    AC_MSG_RESULT(yes)
+    AC_DEFINE(BX_SUPPORT_UINTR, 1)
+    support_cet=1
```

Comment:
Should it be ``support_uintr=1`` instead of ``support_cet=1``?

Reply:
Thanks !